### PR TITLE
Fix conflicts and re-implement eventTopics and accomodationTypes config files placeholders

### PR DESCRIPTION
**Pull request by lukmay**

### CODE REVIEW

File: databrowser/src/config/tourism/accommodationTypes/accommodationTypes.operations.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  create: {
+    rolesAllowed: extendCreateRoles([
+      'accommodationTypesManager',
+      'accommodationTypesCreate',
+    ]),
+  },
+  update: {
+    rolesAllowed: extendUpdateRoles([
+      'accommodationTypesManager',
+      'accommodationTypesUpdate',
```
Comment (gappc) [inline databrowser/src/config/tourism/accommodationTypes/accommodationTypes.operations.ts:25]: Roles are case-sensitive, in this case the correct role names are "AccommodationTypesManager" and "AccommodationTypesUpdate".

File: databrowser/src/config/tourism/accommodationTypes/accommodationTypes.operations.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  readAll: {
+    rolesAllowed: ROLE_READ,
+  },
+  read: {
+    rolesAllowed: ROLE_READ,
+  },
+  create: {
+    rolesAllowed: extendCreateRoles([
+      'accommodationTypesManager',
+      'accommodationTypesCreate',
```
Comment (gappc) [inline databrowser/src/config/tourism/accommodationTypes/accommodationTypes.operations.ts:19]: Roles are case-sensitive, in this case the correct role names are "AccommodationTypesManager" and "AccommodationTypesCreate".

File: databrowser/src/config/tourism/accommodationTypes/accommodationTypes.operations.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  update: {
+    rolesAllowed: extendUpdateRoles([
+      'accommodationTypesManager',
+      'accommodationTypesUpdate',
+    ]),
+  },
+  delete: {
+    rolesAllowed: extendDeleteRoles([
+      'accommodationTypesManager',
+      'accommodationTypesDelete',
```
Comment (gappc) [inline databrowser/src/config/tourism/accommodationTypes/accommodationTypes.operations.ts:31]: Roles are case-sensitive, in this case the correct role names are "AccommodationTypesManager" and "AccommodationTypesDelete".

File: databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  readAll: {
+    rolesAllowed: ROLE_READ,
+  },
+  read: {
+    rolesAllowed: ROLE_READ,
+  },
+  create: {
+    rolesAllowed: extendCreateRoles([
+      'eventTopicsManager',
+      'eventTopicsCreate',
```
Comment (gappc) [inline databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts:19]: Roles are case-sensitive, in this case the correct role names are "EventTopicsManager" and "EventTopicsCreate".

File: databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    ]),
+  },
+  update: {
+    rolesAllowed: extendUpdateRoles([
+      'eventTopicsManager',
+      'eventTopicsUpdate',
```
Comment (gappc) [inline databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts:25]: Roles are case-sensitive, in this case the correct role names are "EventTopicsManager" and "EventTopicsUpdate".

File: databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    ]),
+  },
+  delete: {
+    rolesAllowed: extendDeleteRoles([
+      'eventTopicsManager',
+      'eventTopicsDelete',
```
Comment (gappc) [inline databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts:31]: Roles are case-sensitive, in this case the correct role names are "EventTopicsManager" and "EventTopicsDelete".

**Comment by BlueRyse** [comment]
@gappc Hi Christian thanks for the feedback. I fixed capital letters on roles.

**Review by gappc — APPROVED** [review]

**Comment by gappc** [comment]
Well done, thx

**Pull request merged**